docs/asyncapi.yaml

```diff
@@ -0,0 +1,171 @@
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/Calculation'
+  schemas:
+    Organisation:
+      type: object
+      properties:
+        name:
+          $ref: '#/components/schemas/BusinessName'
+        sbi:
+          $ref: '#/components/schemas/SBI'
+        frn:
+          $ref: '#/components/schemas/FRN'
+        addressLine1:
+          $ref: '#/components/schemas/AddressLine1'
+        addressLine2:
+          $ref: '#/components/schemas/AddressLine2'
+        addressLine3:
+          $ref: '#/components/schemas/AddressLine3'
+        city:
+          $ref: '#/components/schemas/City'
+        county:
+          $ref: '#/components/schemas/County'
+        postcode:
+          $ref: '#/components/schemas/Postcode'
+        emailAddress:
+          $ref: '#/components/schemas/Email'
+        updated:
+          $ref: '#/components/schemas/Updated'
+        type:
+          $ref: '#/components/schemas/Type'
+    Calculation:
+      type: object
+      properties:
+        calculationReference:
+          $ref: '#/components/schemas/CalculationReference'
+        sbi:
+          $ref: '#/components/schemas/SBI'
+        calculationDate:
+          $ref: '#/components/schemas/CalculationDate'
+        invoiceNumber:
+          $ref: '#/components/schemas/InvoiceNumber'
+        scheme:
+          $ref: '#/components/schemas/Scheme'
+        updated:
+          $ref: '#/components/schemas/Updated'
+        fundings:
+          type: array
+          items:
+            $ref: '#/components/schemas/Funding'
+        type:
+          $ref: '#/components/schemas/Type'
+    BusinessName:
+      type: string
+      description: The business name
+      example: 'FFC Ltd'
+    SBI:
+      type: number
+      description: Single Business Identifier
+      minimum: 105000000
+      maximum: 999999999
+      example: 123456789
+    FRN:
+      type: number
+      description: Firm Reference Number
+      minimum: 1000000000
+      maximum: 9999999999
+      example: 1234567890
+    Email:
+      type: string
+      description: The business email address
+      example: farm@farms.com
+    AddressLine1:
+      type: string
+      description: The first line of the address
+      example: 1 The Street
+    AddressLine2:
+      type: string
+      description: The second line of the address
+      example: The Area
+    AddressLine3:
+      type: string
+      description: The third line of the address
+      example: The Town
+    City:
+      type: string
+      description: The fourth line of the address
+      example: The Region
+    County:
+      type: string
+      description: The fifth line of the address
+      example: The County
+    Postcode:
+      type: string
+      description: The postcode
+      example: AB1 2CD
+    Updated:
+      type: string
+      format: date-time
+      description: The date the dataset received from Data Warehouse
+      example: 2022-08-22T16:45:20.891Z
```

Comment:
comes up as empty object on async renderer

changing dashes to slashes, periods fixes this -- question is: do we want to change it to render?

Reply:
Interesting, didn't notice that.

As it's a yaml file can escape special characters by wrapping in quotes `"`.  So have done that.

I would only suggest we change the format of the date if there was a benefit to the consuming services (or service at the moment)